src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -379,6 +380,69 @@ def list_subvolumes(self, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def subvolume_snapshot_mirror_enable(self, **kwargs):
+        ret = 0, "", ""
+        volname = kwargs["vol_name"]
+        subvolname = kwargs["sub_name"]
+        groupname = kwargs["group_name"]
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                with open_group(fs_handle, self.volspec, groupname) as group:
+                    with open_subvol(self.mgr, fs_handle, self.volspec, group, subvolname, SubvolumeOpType.GETPATH) as subvolume:
+                        snap_path = subvolume.snapshot_base_path().decode("utf-8")
+                        subvolume.set_mirrored()
```

Comment:
FYI, this will be used to guard subvolume removal (and can be overridden by `--force`)